tests/test_prep.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+import mne
+import numpy as np
+
+from hypyp import prep
+
+def test_filt():
+    n_channels = 10
+    n_samples = 100000
+    sfreq = 1000
+
+    # create noise signal
+    info = mne.create_info(ch_names=n_channels, sfreq=sfreq, ch_types='eeg')
+    data = np.random.normal(size=(n_channels, n_samples))
+    raw = mne.io.RawArray(data, info)
+    raw_psd = raw.compute_psd().get_data()
+
+    # compare on lowest freq
+    raw_filt_default, = prep.filt([raw])
+    raw_filt_default_psd = raw_filt_default.compute_psd().get_data()
+    assert np.sum(raw_filt_default_psd[:,0]) < np.sum(raw_psd[:,0])
+
+    # compare on highest freq
+    raw_filt, = prep.filt([raw], (2., 10))
+    raw_filt_psd = raw_filt.compute_psd().get_data()
+    assert np.sum(raw_filt_psd[:,-1]) < np.sum(raw_filt_default_psd[:,-1])
+
+
+@pytest.mark.parametrize("fit_kwargs", [
+    dict(method='fastica', fit_params=dict(tol=0.01)), # increase tolerance to converge
+    dict(method='infomax', fit_params=dict(extended=True))
+])
+def test_ICA(epochs, fit_kwargs):
```

Comment:
test moved from `test_stats.py`

- parameterize to test `fastica`
- added amplitude comparison
- added the `plot=False` to avoid launching GUI windows